remoteworkitem/scheduler.go

```diff
@@ -80,17 +102,32 @@ func fetchTrackerQueries(db *gorm.DB) []trackerSchedule {
 func lookupProvider(ts trackerSchedule) TrackerProvider {
 	switch ts.TrackerType {
 	case ProviderGithub:
-		return &GithubTracker{URL: ts.URL, Query: ts.Query}
+		var q string
+		if ts.LastUpdated != nil {
```

Comment:
This if-else block looks repetitive to me, could we compute the value of `q` outside the `switch-case` block and use here?